operator/scripts/utils/cmd.go

```diff
@@ -42,7 +48,35 @@ func ExtractTarball(tarballPath, outputDir string) error {
 	return Execute(cmd)
 }
 
+// getKustomizeVersion returns the version of kustomize or an empty string and an error
+// if there is an issue running the kustomize version command or parsing the command output.
+func getKustomizeVersion() (string, error) {
+	cmd := exec.Command("kustomize", "version", "--short") // will look like: {kustomize/v3.5.4  2020-01-11T03:12:59Z  }
+	output, err := ExecuteAndCaptureOutput(cmd)
+	if err != nil {
+		return "", err
+	}
+
+	versionComponents := strings.Fields(output) // should split as [{kustomize/v3.5.4, 2020-01-11T03:12:59Z, }]
+	if len(versionComponents) != 3 {
+		return "", fmt.Errorf("unexpected output from kustomize version --short: %s", output)
+	}
+
+	versionTokens := strings.Split(versionComponents[0], "/")
+	if len(versionTokens) != 2 {
+		return "", fmt.Errorf("unexpected format of kustomize version --short: %s", output)
+	}
+	return versionTokens[1], nil
+}
+
 func KustomizeBuild(path, output string) error {
+	kustomizeVersion, err := getKustomizeVersion()
+	if err != nil {
+		return err
+	}
+	if requiredKustomizeVersion != kustomizeVersion {
+		return fmt.Errorf("kustomize version mismatch; want: %s, got: %s", requiredKustomizeVersion, kustomizeVersion)
```

Comment:
iiuc justin below mentions that it may be ok to skip mentioning the script. I'd also err on not including that for now. Please let me know if you disagree.